src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -160,7 +161,10 @@ def dangling_inputs(self) -> Entities:
 
 
 class TDRSourceRef(SourceRef[TDRSourceSpec, 'TDRSourceRef']):
-    pass
+
+    @property
+    def typed_id(self) -> TypedSourceId:
+        return TypedSourceId(id=self.id, type=self.spec.type_name)
```

Comment:
If the TDRSourceRef already has the source ID and the source type, why do we need a separate class? Couldn't we pass the TDRSourceRef instance to the methods that currently require an TypedSourceId?

Reply:
Importing `TDRSourceRef` into `terra.py` would create a circular dependency.

Reply:
Decided at PL to move/rename `TDRSourceRef` to avoid this